Rubberduck.Core/UI/UnitTesting/TestExplorerControl.xaml

```diff
@@ -216,6 +216,18 @@
 
                 <Separator />
 
+                <Label Content="{Resx Key=TestExplorer_Filter, ResxName=Rubberduck.Resources.UnitTesting.TestExplorer}" />
+                <TextBox MinWidth="50"
+                         Text="{Binding TestNameFilter, UpdateSourceTrigger=PropertyChanged}"/>
+
+                <Label Content="{Resx Key=TestExplorer_Outcome, ResxName=Rubberduck.Resources.UnitTesting.TestExplorer}" />
+
+                <ComboBox Width="100" 
+                          ItemsSource="{Binding OutcomeFilters, UpdateSourceTrigger=PropertyChanged}"
+                          SelectedItem="{Binding SelectedOutcomeFilter, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
```

Comment:
You should use the `SearchBox` control for this, compare the CodeExplorer:

https://github.com/rubberduck-vba/Rubberduck/blob/6134c87f139b5578a667a3e929054b6e7ec73393/Rubberduck.Core/UI/CodeExplorer/CodeExplorerControl.xaml#L392-L398